docs/book/how-to/manage-zenml-server/upgrade-zenml-server.md

```diff
@@ -85,4 +85,8 @@ Downgrading the server to an older version is not supported and can lead to unex
 The version of the Python client that connects to the server should be kept at the same version as the server.
 {% endhint %}
 
+{% hint style="warning" %}
```

Comment:
there are so many hints stacked together here. would it make sense to make a small paragraph with heading instead?